framework/src/org/checkerframework/common/basetype/BaseTypeVisitor.java

```diff
@@ -957,6 +960,73 @@ public Void visitMethodInvocation(MethodInvocationTree node, Void p) {
     }
 
     /**
+     * A helper method to check that the array type of actual varargs is a subtype of the
+     * corresponding required varargs, and issues "argument.invalid" error if it's not a subtype of
+     * the required one.
+     *
+     * <p>Note it's required that type checking for the each elements of varargs is executed by
+     * caller before or after calling this method.
+     *
+     * @see #checkArguments(List, List)
+     * @param invokedMethod the method type to be invoked
+     * @param tree method or constructor invocation tree
+     */
+    private void checkVarargs(AnnotatedExecutableType invokedMethod, Tree tree) {
```

Comment:
Like `checkArguments` I would make this method protected, in case a subchecker needs to adapt the behavior.